### ISSUE TITLE
Admin login doesn't work because of dead Google OpenID

### ISSUE THREAD

**Issue by pkqk** [theodi/member-directory#360]
Google OpenID is being :sunrise: on 20th of April 2015.

https://developers.google.com/accounts/docs/OpenID#shutdown-timetable

This is currently used for the admin login on the member directory.

It will need to be changed to use OpenID Connect (hashtag brand) or replaced.

**Comment by greenreveller** [comment]
I don't understand :(


Andrea

Andrea Cox
Membership Manager
@andrea_cox_ <https://twitter.com/andrea_cox_> @UKODImembers
<https://twitter.com/UKODImembers>

The Open Data Institute <http://www.theodi.org/>
Tel +44 (0) 7855 456 002

**Comment by pkqk** [comment]
The way authorization to Administration is done currently is by using your google account (and checking it's an `@theodi.org` one).

Google is shutting down this ability, we can either change it to use the new Google service (depending on how simple that is) or just set it to have a password.

Though I just realised since #303 is still a problem, you may not be using the admin much, or are you just hacking around that problem.

**Comment by greenreveller** [comment]
Ah, ok. thanks for clarifying. I'm still hacking around but deleting the 443


Andrea

Andrea Cox
Membership Manager
@andrea_cox_ <https://twitter.com/andrea_cox_> @UKODImembers
<https://twitter.com/UKODImembers>

The Open Data Institute <http://www.theodi.org/>
Tel +44 (0) 7855 456 002

**Comment by greenreveller** [comment]
Ah, i think we have hit crunch point: https://support.google.com/accounts/answer/6206245?p=openid&rd=1. As @pkqk mentioned we need to migrate to OpenID Connect: https://developers.google.com/identity/protocols/OpenID2Migration#shutdown-timetable
@Floppy @pezholio

**Comment by jordelver** [comment]
This is now fixed and ready for testing @greenreveller

**Comment by greenreveller** [comment]
Link works and I can make Directory Entry text changes via this link (both accurately reflected on https://directory.theodi.org/members and in Capsule) 

I've tested adding a new account email address to the Syngenta account (from derek.scuffell@syngenta.com to sarah.targett@syngenta.com). I would expect to see a new welcome email go out to Sarah in the next couple of hours (though dependent on #459).

**Comment by jordelver** [comment]
Ok. Great. If any further issues come up, can you log them as a new issue, please? This one just covers the login problem and that sounds like it's fixed now?

**Comment by greenreveller** [comment]
Yes sure - adding new issue now. Great to have OpenAuth sorted :+1:
